_posts/2019-01-16-lecture-02.md

```diff
@@ -0,0 +1,459 @@
+* $$x_3 \rightleftharpoons  x_1 \rightleftharpoons x_4$$: due to 'common cause' structure, this trail is blocked if we condition on $$x_1$$. Therefore we get $$(x_3 \perp \{x_4\} \vert x_1), (x_3 \perp \{x_4\} \vert \{x_1, x_2\})$$
+
+### Trails of length 4 (only $$x_2 \rightleftharpoons  x_3 \rightleftharpoons x_1 \rightleftharpoons x_4$$):
+* due to 'common effect' structure $$x_2 \rightarrow x_3 \leftarrow x_1$$, this trail is blocked as long as we do not condition on $$x_3$$. Therefore we get $$(x_2 \perp \{x_4\}),  (x_2 \perp \{x_4\} \vert x_1)$$.
+* due to 'common cause' structure $$x_3 \leftarrow x_1 \rightarrow x_4$$, this trail is blocked if we condition on $$x_1$$. Therefore we get $$(x_2 \perp \{x_4\} \vert x_1), (x_2 \perp \{x_4\} \vert \{x_1, x_3\})$$
+
+### Full $$I(\mathcal{G})$$ 
+Putting the above together we have the following independencies.
+
+$$\begin{aligned}
+I(\mathcal{G}) 
+&= \{(x_2 \perp \{x_1\}), (x_2 \perp \{x_1\} \vert x_4), \\
+&(x_3 \perp \{x_4\} \vert x_1), (x_3 \perp \{x_4\} \vert \{x_1, x_2\}), \\
+&(x_2 \perp \{x_4\}),  (x_2 \perp \{x_4\} \vert x_1),  (x_2 \perp \{x_4\} \vert \{x_1, x_3\}) 
+\end{aligned}$$
+
+## The Equivalence Theorem 
+
+For a graph $$G$$, Let $$D_1$$ denotes the family of all distributions that satisfy $$I(G)$$. Let $$D_2$$ denotes the family of all distributions that factor according to $$G$$. Then $$D_1 \equiv D_2$$, which can be expressed as 
+
+$$ P(X)=\prod_{i=1:d}P(X_i \vert X_{\pi_i}) $$.
+
+This means separation properties in the graph imply independence properties about the associated variables.
+
+## Conditional Probability Tables (CPTs)
+
+This is an example with discrete probabilities.
+
+<img src="{{ '/assets/img/notes/lecture-02/CP_1.png' | relative_url }}" />
+
+
+## Conditional Probability Density Func. (CPDs) 
+
+The probabilities can also be a continuous function, e.g. the Gaussian distribution.
+
+<img src="{{ '/assets/img/notes/lecture-02/CP_2.png' | relative_url }}" />
+
+## Summary of BN
+
+* Conditional independencies imply factorization.
+
+* Factorization according to $$G$$ implies the associated conditional independencies.
+
+## Soundness and Completeness of D-separation 
+
+* Soundness:
+
+    Theorem: If a distribution $$P$$ factorizes according to $$G$$, then $$I(G)\subseteq I(P)$$ 
+* "Completeness":
+
+    "Claim": For any distribution $$P$$ that factorizes over $$G$$, if $$(X \perp Y \vert Z) \in I(P)$$, then $$ \text{d-sep}_{\mathcal{G}} (X; Y \vert Z) $$ )
+
+  * Actually, the "Completeness" is not holding true all the time. "Even if a distribution factorizes over G, it can still contain additional independencies that are not reflected in the structure."
+  
+  * Example:
+
+    Given graph $$ A->B $$, and a conditional probability table,
+    <img src="{{ '/assets/img/notes/lecture-02/CPT.png' | relative_url }}" />
```

Comment:
This image looks really blurry and doesn't fit into the text. Can you improve the resolution and put the table into the `<div class="l-gutter">...</div>`?

Reply:
Hi, Mauran, how to change it? I reckon I can't do it in this repo, right?

Reply:
Just push changes to the master branch of @ceesu's fork. The changes will appear here automatically.